examples/SLAM/README.md

```diff
@@ -0,0 +1,20 @@
+# SLAM (World tracking) camera demo
+![Preview](previews/SLAM-example.webp?raw=true "Dynamically generated mesh")
+Runs an AR session using WebXR Device AP where available with a fallback to 8th-wall implementation.
+
+To successfully run in 
+- make sure `API_TOKEN_XR8` is defined in the index.html file
+```
+<script>
+  const API_TOKEN_XR8 = "your 8thwall project API token";
+</script>
+```
+
+- Make sure const `WEBXR_REQUIRED_FEATURES` and `WEBXR_OPTIONAL_FEATURES` are defined in the index.html file
+```
+<script>
+      const WEBXR_REQUIRED_FEATURES = [{{ webxrRequiredFeatures }}];
+      const WEBXR_OPTIONAL_FEATURES = [{{ webxrOptionalFeatures }}];
```

Comment:
We could put these into the upcoming `/* wle:auto-constants:(start|end) */` section, if that's useful?
Ideally the user doesn't have to do any index.html modifications to use the framework.

Reply:
Would be great to include the auto-constants!